### PR TITLE
513 merge ronin

### LINKED ISSUE #513: Merge Ronin into MA.com and improve UI
As a future student (and Hiring Partner),
I want to land on a single page that describes what MA is with a division in pricing and Value Proposition

Onboarding and Careers seem to agree it makes sense to have the whole marketing effort pointing to a single website rather than two different ones

### PR DESCRIPTION
**Pull request by sjmog**
Closes #513

### CODE REVIEW

**Comment by sjmog** [comment]
:warning: :warning: :warning: :warning: :warning: :warning: :warning: :warning: :warning: 

DO NOT MERGE until copy has been updated!

:warning: :warning: :warning: :warning: :warning: :warning: :warning: :warning: :warning:

File: source/partials/_benefits_table.html.haml [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+      %h2 Remote or Onsite?
     .centered-row
       %table
+        %thead
+          %tr
+            %td Ronin
+            %td Makers Academy
         %tbody
           %tr
+            %td £2,400
```
Comment (danldb) [inline source/partials/_benefits_table.html.haml]: Did we lower the price?

**Comment by sjmog** [comment]
# Mobile

screen shot 2016-04-07 at 15 21 54
screen shot 2016-04-07 at 15 22 04
screen shot 2016-04-07 at 15 22 11
screen shot 2016-04-07 at 15 22 18
screen shot 2016-04-07 at 15 22 25
screen shot 2016-04-07 at 15 22 33
screen shot 2016-04-07 at 15 22 39
screen shot 2016-04-07 at 15 22 46
screen shot 2016-04-07 at 15 22 52

File: source/partials/_benefits_table.html.haml (hunk 1 above)
Comment (sjmog) [inline source/partials/_benefits_table.html.haml]: I've noted this as a content update so we can get eyes on the placeholder stuff I put in ;)

**Comment by danldb** [comment]
Looks nice! Love the 'onsite or remote' table. Currently No option for Ronin on the application form. No differentiation in discounts Ronin vs Onsite (i.e. 10% discount for early payment, discount for women).

**Comment by danldb** [comment]
Also FAQ still has links to Ronin.

**Comment by sjmog** [comment]
# Desktop

screen shot 2016-04-07 at 15 25 32
screen shot 2016-04-07 at 15 25 38
screen shot 2016-04-07 at 15 25 48
screen shot 2016-04-07 at 15 25 54
screen shot 2016-04-07 at 15 25 59
screen shot 2016-04-07 at 15 26 05
screen shot 2016-04-07 at 15 26 11

**Comment by nikeshashar** [comment]
@sjmog - how would the apply button differ? would we also have to merge the application forms or would we have two CTA's in the header?

**Comment by nikeshashar** [comment]
@danldb FAQ's are being moved to GrooveHQ for everything once the button is styled. It will be a searchable system so we can help the customers quicker.

**Comment by Arfah** [comment]
I'm really against putting the price in the heading. It will effect things dramatically and lead to a huge bounce rate and will make people exit very quickly.

I'd rather do the soft sell then have the price at the bottom.

I can't remember putting email at the bottom but maybe I did. I would prefer it at the top.

I like the comparison table but maybe we can add that a big higher to show we have two products.

The price at the top is just a big no no...

Also will their be links from other pages like curriculum etc going to ronin.makersacademy. Not sure what the discussion was...

**Comment by Arfah** [comment]
And yes application...

I envision one application form but if you select ronin the extra questions opens up

File: source/partials/_email-capture.html.haml [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+  .container
+    .centered-row
+      %h4.leader Keep 
+      %h2 up-to-date
+      %p.lead Sign up for our newsletter and get your free eBook: Learn Ruby in 15 minutes.
+    .email-capture-row
+      %form.email-capture
+        %label{for: "newsletter-email"}
+        %input{type: "email", id: "newsletter-email", placeholder: "Your Email Address", required: true, 'data-segments': locals.fetch(:segments, "General") }
+        %input.button.button--horizontal{type: "submit", value: locals.fetch(:button_text, "Send my e-book") }
```
Comment (danldb) [inline source/partials/_email-capture.html.haml]: This partial is also used on the Graduates page - it looks really weird over there.

**Comment by sjmog** [comment]
Let's chat more about this in the upcoming week, but these changes are all from the mockups we drew from the perspective of our customers after our Core Model workshop 18th February. My job has been to take the mockups and convert them to the final merge, which was agreed as below:

img_0241

If we're concerned about the scale of some of the changes, let's roll them out gently using A/B testing so we can flag up any major problems.

To remind us, when we were forced to prioritise using post-it notes, things like the newsletter _actually_ wound up very low. A meta-problem we identified was that we prioritise everything too highly, and don't discriminate enough. For instance, the newsletter is actually much less important for us and for customers than knowing the price:

img_0235
img_0236
img_0237

If possible, let's avoid reinventing the wheel and going back to the drawing board on this one.

**Comment by sjmog** [comment]
@nikeshashar yes, we will need to update the application form to allow Ronin applications on the same form.

**Comment by jordanpoulton** [comment]
Let's get together to hash this out?!? Wed at 10am?

**Comment by rubenkostucki** [comment]
I think an ESSENTIAL part of this page has to be around the table. The table cannot look like this in my opinion and it'll be the main focal point for the differentiation. I think let's steal from the best pricing pages and show it like the squarespace one on http://blog.hubspot.com/marketing/pricing-page-examples

I like the fact that there's a short FAQ right away under. It's also clear what the differences are.

And rather than put the price at the top, let's just have one landing page which explains there's a remote and onsite version and the pricing page, design it like squarespace.

Or else, do it more like the image below (but a MUCH nicer design): 

description | ronin V or X | onsite V or X

obox-pricing-chart9

**Comment by rubenkostucki** [comment]
I agree with Arfah about the pricing at the top. What I'd suggest is a few words under each:

Onsite
In our London Campus

Remote
from the comfort of your home

File: source/partials/_benefits_table.html.haml (hunk 1 above)
Comment (pitchinvasion) [inline source/partials/_benefits_table.html.haml]: Shall we extract the price to a config so it's easily changed?

**Comment by pitchinvasion** [comment]
I think it would make sense to merge the application form, it shouldn't be too much work to do this. Otherwise we'd have to have some sort of page in between the form which is a bad idea

**Comment by jordanpoulton** [comment]
Actually my feedback here isn't anything new, just adding to previous statements...

1) I agree that we shouldn't mention the price until the bottom. It's better for sales... 
2) I agree with Ruben around styling the table - how should we go about designing this?

Other than that, we just need to make sure that the copy is fresh & on brand etc, but this is a great start Samm. 

How do we go forward?

**Comment by nikeshashar** [comment]
I'd request to make the dates more prominent and split them visually somehow to differentiate Ronin / MA cohorts. Maybe using the logo's of each ?

**Comment by sjmog** [comment]
I've updated the merge by removing the up-front costs and replacing as Ruben suggests. I've also moved the statistics to a data file, and am going to use the Ronin/MA logos to differentiate the table.

The problem with a 'ticks' table, which I think @rubenkostucki is angling for, is that it requires 3 columns, which doesn't work on a mobile view (note that most 'mobile tables' for comparison are scrolly and generally suck). So I suggest that we stick with what we've got here for the time being.

@pitchinvasion we do need to update the application form to work for both Ronin and MA if this is the direction we're heading in.

Once we settle on this homepage design I'll make sure the rest of the site flows properly as @danldb points out.

**Comment by sjmog** [comment]
Benefits table restyled! Let me know what you think now.

**Comment by rubenkostucki** [comment]
1. It would be nice from now on to "mark" Ronin on the graduates's profiles so it's easy for people to see that Roniners do get jobs (and Arfah, place a few Ronin people early in the carousel). An idea is add the Ronin logo somewhere on the image like a 'stamp of approval'?

2. I still think the table is not good enough @sjmog - I'm happy to train and come up with a few ideas on how to improve it. The important part is having a nice way to show the differences and this looks very bland IMO

3. We'd of course have to re do a lot of pages like /payment/ but I guess that's not the point of this particular ticket, is it? /curriculum/ would also have to vary a little as we don't provide meditation and yoga like we do onsite.

The differentiations in 2. with the elements I mentioned in 3. is the key to making this page successful IMO so worth making it really awesome

**Comment by jordanpoulton** [comment]
Some more feedback from a friend of mine who works in UX:

1) In the first partial, remove the prices, replace with 'find out more' that takes the user to the current Makers / Ronin landing pages

2) Pare down the info in the Ronin vs Makers comparison to the 'top 3 differences' then have a button to click through to a more detailed comparison (where we will have space for the 3rd option - 1-1 tuition @ £80k

3) Rather than have the 'hire graduates' in with 'Payment options' and 'apply', consider giving it its own partial?

4) Maybe reiterate the 1st partial at the bottom (with different headline/body text but the same 2 buttons suggested in (1)), below apply, so if people aren't ready to apply yet, then can click the button to find out more about MA or about Ronin, whichever they're more interested in.

**Comment by rubenkostucki** [comment]
Lots of it makes sense actually. I still think the table is a trouble the
way it is now. We might have to use colours as well to make it look better
and clearer. Using crosses when not relevant for Ronin would be a good idea
too IMO
On Thu, 14 Apr 2016 at 16:32, Jordan Poulton <notifications@github.com>
wrote:

> Some more feedback from a friend of mine who works in UX:
>
> 1) In the first partial, remove the prices, replace with 'find out more'
> that takes the user to the current Makers / Ronin landing pages
>
> 2) Pare down the info in the Ronin vs Makers comparison to the 'top 3
> differences' then have a button to click through to a more detailed
> comparison (where we will have space for the 3rd option - 1-1 tuition @ £80k
>
> 3) Rather than have the 'hire graduates' in with 'Payment options' and
> 'apply', consider giving it its own partial?
>
> 4) Maybe reiterate the 1st partial at the bottom (with different
> headline/body text but the same 2 buttons suggested in (1)), below apply,
> so if people aren't ready to apply yet, then can click the button to find
> out more about MA or about Ronin, whichever they're more interested in.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/makersacademy/website/pull/588#issuecomment-209971823>
>
-- 
___________________________

Ruben Kostucki,
COO

+44 774 829 9962 |  www.makersacademy.com

**Comment by Arfah** [comment]
Beyond the homepage there are small copy changes that need to be made.

I've started to rework things in here: https://docs.google.com/a/makersacademy.com/spreadsheets/d/1gV7pDlP_jbwMslpnBT9547Si5exjLFHvR5R3n7dNrBQ/edit?usp=sharing

Also the need to highlight ronin graduates is also on the list of priorities for me. So I've reached out to them to start gathering the case studies.

Me and sam have come up with an idea that we'll photoshop their face on a laptop of a body holding the laptop to show a "ronin" student so you'll know who is online through this creative picture.

The other thing is... the table is better but at a glance it doesn't look much different. On my landing pages for example the table partial will be very important to show they can do the course in two diff ways.

**Comment by sjmog** [comment]
I am considering a 'table change' more in the style of the below (obv flatter). The tricky bit is mobile.

screen shot 2016-05-10 at 10 52 53

@rubenkostucki I'll give you a gander once I've had a pop.

**Comment by sjmog** [comment]
@rubenkostucki at the moment the [comparison spreadsheet](https://docs.google.com/spreadsheets/d/1LS1JISqnfflZyTVrArFn-JF_lrgyj1onPPkCtmQYPm0/edit#gid=0) is bit busy - let's aim for ten comparison points, and we can always have a button that shows a more fully-fledged comparison. What are the top ten comparison points?

**Comment by sjmog** [comment]
@jordanpoulton I'm going through the site and removing some extra stuff that seems to have been bodged across from the Ronin site - for example, the graduates page now contains too many video partials, and it crashes the page (as well as mussing up the page flow).

**Comment by sjmog** [comment]
We need to make sure the graduate slider/video stuff renders ten graduates tops (no-one will click through all of them and loading all the people/photos is killing the load time)

**Comment by sjmog** [comment]
We should select ten graduates who:

- Are distributed across time (ideally, one per cohort for the last ten cohorts or so)
- Demonstrate of the diversity we have here at Makers Academy
- Have the highest-quality photographs
- Have the highest-quality stories

When adding to the website, we should aim to keep that number at ten (i.e. one in means one out)

**Comment by nikeshashar** [comment]
@sjmog we'd need to find a way to keep the data on the site and display only the 10 we want. Landing pages use people that would otherwise be removed under this. 

Plus I use the site during interviews to demonstrate people who are the most likely match to the person I'm interviewing.

**Comment by sjmog** [comment]
Very fair @nikesh - a chat with Marketing just a moment ago we confirmed that we'll:

- Try to think of a cleverer way to merge Ronin and Onsite content (e.g. vlogs)
- Cut down to 10 people on the homepage (5 recent and 5 'best of all time' that meet the criteria above)
- Show everyone on the Graduates page
- Move the 'One Year On' content to be accessible from a button on each graduate in the graduates slider on the graduates page, thus removing it from the main graduates page flow

This should keep our trim design while ensuring our page load speed doesn't drop too much.

**Comment by pitchinvasion** [comment]
@sjmog just to confirm when are you planning on getting this out? As my ticket to merge the application forms is blocking this, although I can hopefully do that on Thursday

**Comment by pitchinvasion** [comment]
Also this is story feeling a little epic, is there a plan to break this down into smaller stories?

**Comment by sjmog** [comment]
@pitchinvasion I hear you, but we're almost there with the merge - the only outstanding factor is the table.

Can everyone here please check out the desktop version of the comparison table? I'll start building a mobile one once we're good with it.

You can now add/remove benefits in a data file called 'comparisons'.

**Comment by pitchinvasion** [comment]
Fair enough how about the planned release? When do I need to merge the application forms by?

**Comment by sjmog** [comment]
@pitchinvasion planned go-live was Thursday, but we can hold until the application forms are merged. When do you think you could do that by?

**Comment by sjmog** [comment]
I've now added a mobile variant of the comparisons table. For my sins, I decided to simply different HTML for the mobile table (specifically, to avoid some dodgy cross-browser handling of Flexbox).

Both tables draw from one source of truth: the `comparisons.yml`, so updating info there will update both kinds of tables.

Please test it and see what you think.

Only waiting on the merged form now @pitchinvasion

**Comment by sjmog** [comment]
Some more tidying done and merged with master effectively, ready to go!

**Comment by spike01** [comment]
* The asterisk after £2500 on the comparison chart would usually imply that there's somewhere that explains why it's there - not that the figure itself is clickable. 

* Ronin is simply referred to as "remote" pretty much everywhere except for "our prices for Makers Academy and Ronin are changing". Is this is a rebrand or not? Might be confusing for people who are new to us and don't know the difference.

**Comment by Arfah** [comment]
Doing a final sanity check 
tweeked some copy
but just realised a major thing is missing in the merge: http://www.makersacademy.com/payment/

^ We no longer have a link to this. This was always the bottom in the apply now partial and was the only way to get to payments.

Kinda important to bring this back in.

Also noticed when you click "Apply Remote" https://apply.makersacademy.com/remote/application link is broken - It prob needs to be changed from ronin to remote...

**Comment by sjmog** [comment]
@Arfah nice catches! Both are fixed now, there was a namespace conflict (Middleman was rendering the `apply.html.haml` page into the places we asked for the partial `_apply` @pitchinvasion )

**Comment by nikeshashar** [comment]
@sjmog we need to add Remote terms and conditions to the new site from the old Ronin site.

**Comment by nikeshashar** [comment]
@sjmog @pitchinvasion Remote terms added to the footer and titles updated here : https://github.com/makersacademy/website/pull/612 (PR to this branch)

**Pull request merged**